src/pages/learn/index.tsx

```diff
@@ -0,0 +1,943 @@
+            <p>
+              Ethereum has led to the creation of new products and services that
+              can improve different areas of our lives. We're still in the early
+              stages but there's a lot to be excited about.
+            </p>
+            <CardGrid>
+              <Card
+                title="Decentralized Finance"
+                description="Explore an alternative financial system that is built without banks and is open to anyone."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.financeTransparent)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/defi/">What is DEFI?</ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Stablecoins"
+                description="Stablecoins are cryptocurrencies designed to stay at a fixed value of a dollar."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.stablecoins)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/stablecoins/">
+                    What are stablecoins?
+                  </ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Non-fungible tokens"
+                description="Non-fungible tokens represent ownership of unique items."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.infrastructureTransparent)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/nft/">What are NFTs?</ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Decentralized autonomous organizations"
+                description="Organizations that run without the need for coordination by a central entity."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.dao)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/dao/">What are DAOs?</ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Decentralized applications"
+                description="Are creating a new digital economy of peer-to-peer services."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.developersEthBlocks)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/dapps/">Explore dapps</ButtonLink>
```

Comment:
Shouldn't this follow the same 'what is ...' pattern as the other cards?

```suggestion
                  <ButtonLink to="/dapps/">What are dapps?</ButtonLink>
```

<img width="1147" alt="Screenshot 2022-07-07 at 16 08 51" src="https://user-images.githubusercontent.com/62268199/177807953-8b3a1b36-0d4d-4e8f-b64f-17a4194074bc.png">